tutorials/python/python-type-driven-development/tutorial.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Type-Driven Development in Python
+published: 2019-08-01
+tags: Python, typing, types, tdd
+language: python
+author-name: Carlos Villavicencio
+twitter-profile: po5i
+github-profile: po5i
+description: Learn how to improve the software quality and readability in our Python code by using the built-in typing system and complementing with other software development processes like type-driven development and test-driven development.
+---
+
+_Type-Driven Development_ is a simple approach that states to add static types to our code that are aligned with the logic. It brings many benefits such as increasing the performance, accuracy, testability, readability, and extensibility of your code by taking advantage of the language’s type system. There are many books that you can use to learn how to achieve this, for example, [“Type-Driven Development with Idris”][tdd-idris] and extrapolate these practices to other languages.
```

Comment:
I think we need to revise the first sentence. Perhaps something like... Type-Driven Development is a technique in which we add static types to our code in order to... 

I have to read more of this post to know what a proper lead-in would be.